schemas/file.yml

```diff
@@ -33,13 +33,30 @@
   - name: directory
     level: extended
     type: keyword
-    description: Directory where the file is located.
+    short: Directory where the file is located.
+    description: >
+      Directory where the file is located. It should include the drive letter,
+      when appropriate.
     example: '/home/alice'
 
+  - name: drive_letter
+    level: extended
+    type: keyword
+    ignore_above: 1
+    short: Drive letter where the file is located.
+    description: >
+      Drive letter where the file is located.
```

Comment:
I think the docs should mention this is a Windows concept to alleviate any confusion on the part of implementers working on non-Windows file info.

Reply:
@andrewkroh Done.

Do you see any issue with stripping off the `:`?

Reply:
No issue. It doesn't provide any additional information.